src/main/java/io/stargate/sgv2/jsonapi/service/shredding/Shredder.java

```diff
@@ -382,7 +382,7 @@ private void validateObjectValue(String referringPropertyName, JsonNode objectVa
       final int propCount = objectValue.size();
       if (propCount > limits.maxObjectProperties()) {
         throw ErrorCode.SHRED_DOC_LIMIT_VIOLATION.toApiException(
-            "number of properties an indexable Object ('%s') has (%d) exceeds maximum allowed (%s)",
+            "number of properties an indexable Object (field '%s') has (%d) exceeds maximum allowed (%s)",
```

Comment:
Minor nitpick but I'd change "field" -> "property" for consistency here, because you already have "number of properties". And likely in the other error messages as well. Occasionally using "field" and "property" interchangeably is fine, but best to stay consistent in the same sentence.